### PR TITLE
feat: like/dislike functionality

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes

- Like/Dislike functionality
image

### CODE REVIEW

File: src/pages/VideoDetails/VideoDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    const {state: {token}} = useAuth()
+    const { likeVideo, setLikeVideo} = useLike()
+    const currentVideoPlaying = videos.find(item => item._id === videoId)
+
+    const addToLiked = async (video) => {
+      try {
+        const resp = await axios.post('/api/user/likes' , {video}, {
+          headers: {authorization: token}
+        })
+        console.log(resp);
```
Comment (tushargoyal812) [inline src/pages/VideoDetails/VideoDetails.jsx:23]: remove console

**Review by tushargoyal812 — COMMENTED** [review]
code is clean and to the point.
variable naming is good.
good to merge

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: src/pages/VideoDetails/VideoDetails.jsx (hunk 1 above)
Comment (sharmayatendra) [reply src/pages/VideoDetails/VideoDetails.jsx:23]: ok

**Pull request merged**